custom/icds_reports/utils/aggregation_helpers/__init__.py

```diff
@@ -26,6 +29,14 @@ def get_agg_child_temp_tablename():
     return 'tmp_agg_child_health_5'
 
 
+def get_prev_agg_tablename(table_alias):
+    return f'{table_alias}_prev'
+
+
+def is_current_month(month):
+    return month == transform_day_to_month(date.today())
```

Comment:
Are we sure about the robustness of this during the month start. Like if the agg is kicked of on 31st night. and when it gets to agg_Awc it is already next month. What is the behavior we expect then?

Reply:
That is a great question. I think I may have to rethink this one

Reply:
On second pass, I don't actually think there is a downside to this. Worst case it uses the temp tables, rather than the live UCRs, but those will exist and contain correct data anyway, so that sounds fine.